### PR TITLE
Chore/scopechange/styling

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: AB#FILL_IN_YOUR_ISSUE_ID

## Checklist:

-   [x] I have performed a self-review of my own code.
-   [ ] I have linked my DevOps task using the AB# tag.
-   [ ] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/apps/ScopeChangeRequest/Api/PCS/Types/Tag.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    StatusDescription: string;
+    TagFunctionCode: string;
+    TagFunctionDescription: string;
+    CommPkgNo: string | null;
+    McPkgNo: string | null;
+    PurchaseOrderNo: string;
+    CallOffNo: string;
+    ProjectDescription: string;
+    Sequence: null;
+    MountedOnTagNo: null;
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Api/PCS/Types/Tag.ts:17]: unknown | null

File: src/apps/ScopeChangeRequest/Api/PCS/Types/area.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export interface Area {
+    Id: number;
+    ParentId: null;
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Api/PCS/Types/area.ts:3]: unknown | null

File: src/apps/ScopeChangeRequest/Api/PCS/Types/discipline.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export interface Discipline {
+    Id: number;
+    ParentId: null;
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Api/PCS/Types/discipline.ts:3]: unknown | null

File: src/apps/ScopeChangeRequest/Components/STID/Components/StidDocument.tsx [hunk 4]
```diff
@@ -2,17 +2,33 @@ import { tokens } from '@equinor/eds-tokens';
...
+        [QueryKeys.Document, `${docNo}`],
+        () => getDocumentById(docNo, 'JCA'),
+        {
+            staleTime: Infinity,
+            cacheTime: Infinity,
+        }
+    );
+
+    function transformIsoDate(date: string | undefined) {
+        if (!date) return '-';
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/STID/Components/StidDocument.tsx:28]: move to date format file

File: src/apps/ScopeChangeRequest/Hooks/useApiActionObserver.ts [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+    useEffect(() => {
+        setIsFetching(mutating > 0);
+    }, [mutating]);
+
+    useEffect(() => {
+        setIsFetching(queriesFetching > 0);
+    }, [queriesFetching]);
+
+    const [isFetching, setIsFetching] = useState<boolean>(false);
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Hooks/useApiActionObserver.ts:16]: move

**Review by jaysencpp — APPROVED** [review]

**Pull request merged**